ee/hogai/graph/inkeep_docs/nodes.py

```diff
@@ -32,6 +33,7 @@ class InkeepDocsNode(RootNode):  # Inheriting from RootNode to use the same mess
     def node_name(self) -> MaxNodeName:
         return AssistantNodeName.INKEEP_DOCS
 
+    @billable(is_billable=False)
```

Comment:
Question: In `ee/hogai/utils/test/test_billing.py` you test `inkeep_docs is billable (not in excluded list)`, but here it's false, how does it work?